src/ServerProviders/Nginx/Plugins.ServerProviders.Nginx/NginxManager.cs

```diff
@@ -212,7 +223,7 @@ public async Task<List<BindingInfo>> GetBindings()
             }
 
             // find config files, read nginx main config, then included config files
-            var primaryConfigPath = Path.Combine(_configPath, _primaryConfigFile).Replace("\\", "/");
+            var primaryConfigPath = _isWindows ? Path.Combine(_configPath, _primaryConfigFile) : Path.Combine(_configPath, _primaryConfigFile).Replace("\\", "/");
```

Comment:
I can't remember if this Replace is really required or not or why it was there! As the Replace will do nothing in a proper unix/linux path and windows optionally also understands `/` as a path separator, should it perhaps still run for both anyway? e.g. if the user supplies the wrong path separator in their config when targeting linux.

Reply:
We should also consolidate this logic into one method as it appears in multiple places.

Reply:
I've added a new method for this logic, but we can remove it if you feel that it is not needed